CHANGELOG.md

```diff
@@ -111,6 +111,10 @@
   ([PR #710](https://github.com/jasmin-lang/jasmin/pull/710);
    fixes [#709](https://github.com/jasmin-lang/jasmin/issues/709)).
 
+- Fix bug in speculative constant time checker
+  ([PR #723](https://github.com/jasmin-lang/jasmin/pull/723);
+   fixes [#724](https://github.com/jasmin-lang/jasmin/issues/724)).
+
```

Comment:
This entry might be merged with the one announcing the SCT checker.

Reply:
Yes, and in that case, it make no sens to have it.

Reply:
You may keep the link to this PR.

Reply:
I think it is now ready, excepted the horrible history.